src/tools/how_to_mock_slice.ts

```diff
@@ -10,8 +10,8 @@ import { SharedSlice } from "@prismicio/types-internal/lib/customtypes"
 
 import { telemetryClient } from "../server"
 
-export const how_to_upsert_mock_slice = tool(
-	"how_to_upsert_mock_slice",
+export const how_to_mock_slice = tool(
+	"how_to_mock_slice",
```

Comment:
i thought i had changed this, maybe the ai put it back before committing in a previous pr